docs/docs/integrations/chat/snowflake.ipynb

```diff
@@ -94,13 +94,13 @@
     "    top_p=0.95,\n",
     "\n",
     "    # Specify your Snowflake Credentials\n",
-    "    account=\"YOUR_SNOWFLAKE_ACCOUNT\",\n",
-    "    username=\"YOUR_SNOWFLAKE_USERNAME\",\n",
-    "    password=\"YOUR_SNOWFLAKE_PASSWORD\",\n",
-    "    database=\"YOUR_SNOWFLAKE_DATABASE\",\n",
-    "    schema=\"YOUR_SNOWFLAKE_SCHEMA\",\n",
-    "    role=\"YOUR_SNOWFLAKE_ROLE\",\n",
-    "    warehouse=\"YOUR_SNOWFLAKE_WAREHOUSE\"\n",
+    "    snowflake_account=\"YOUR_SNOWFLAKE_ACCOUNT\",\n",
```

Comment:
The attributes inside of the chat model namespace should not include the `snowflake_` prefix.